src/Uno.UI/UI/Xaml/Documents/UIElementTextHelper.wasm.cs

```diff
@@ -107,7 +107,15 @@ internal static void SetFontSize(this UIElement element, object localValue)
 
 		internal static void SetMaxLines(this UIElement element, object localValue)
 		{
-			// Not available yet
+			if (localValue is UnsetValue)
+			{
+				element.ResetStyle("display", "-webkit-line-clamp", "webkit-box-orient", "overflow");
```

Comment:
You should not play with the `overflow` style, because it's already managed by the layouting engine.  Do you really need this for the solution to work?

Reply:
Looks like it works with default overflow. However, with overflow set to show it doesn't work (since the line never wraps). Not sure if that can be considered a problem though since that seems like a pretty straightforward functionality